src/otx/algo/action_classification/heads/x3d_head.py

```diff
@@ -0,0 +1,102 @@
+# Copyright (C) 2024 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+# Copyright (c) OpenMMLab. All rights reserved.
+
+"""X3D head implementation."""
+from __future__ import annotations
+
+from torch import Tensor, nn
+
+from otx.algo.action_classification.heads.base_head import BaseHead
+from otx.algo.utils.weight_init import normal_init
+
+
+class X3DHead(BaseHead):
+    """Classification head for I3D.
+
+    Args:
+        num_classes (int): Number of classes to be classified.
+        in_channels (int): Number of channels in input feature.
+        loss_cls (dict or ConfigDict): Config for building loss.
```

Comment:
```suggestion
        loss_cls (nn.Module): Config for building loss.
```
It seems typo

Reply:
fixed at 36e79c2fc6f88460f922d3eec9b6fe1e01475db0